src/base/bittorrent/torrentinfo.h

```diff
@@ -66,6 +66,7 @@ namespace BitTorrent
 
         static TorrentInfo load(const QByteArray &data, QString *error = nullptr) noexcept;
         static TorrentInfo loadFromFile(const QString &path, QString *error = nullptr) noexcept;
+        void saveToFile(const QString &path);
```

Comment:
Looks like this function can be `const`.
And I also noticed that it didn't access any of the class variables...

Reply:
Ok.